tests/unittests/torch_dataset_tests.py

```
@@ -167,6 +210,250 @@ def test_skip_failures(self):
+        # an error when calling get_item.
+        n_per_sample = (2, 2)
+        dataset = _make_detection_dataset(n_per_sample)
+
+        # Clear the `label` on the first detection of the first sample
+        sample = dataset.first()
+        sample["ground_truth"].detections[0].label = None
+        sample.save()
+
+        class RequireLabel(GetItem):
+            @property
+            def required_keys(self):
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `required_keys`

(ANN202)